tests/conftest.py

```diff
@@ -329,7 +340,7 @@ def sequential_ids():
     """Sequential uuids for files."""
```

Comment:
You can add `scope="session"` to the `sequential_ids` fixture as it never changes.